### DISCUSSION TITLE
How do you deploy your packages to wordpress.org?

### DISCUSSION THREAD

**Discussion by binarykitchen** [WordPress/gutenberg#51113] [Developer Experience]
Hello all,

Nice project here, I'm impressed. Just curious, how do you deploy packages to WordPress.org itself? Not npm.

Because WordPress requires a SVN repository. How do you sync with them for deploying changes, when this is GitHub?

Thanks for any pointers
Michael

**Comment by t-hamano** [comment]
Hi @binarykitchen,

What you want to know is how the plugin is generated from this Gutenberg project and how it is released to the WordPres.org plugin directory?

If so, you may want to analyze [this GitHub Actions workflow file](https://github.com/WordPress/gutenberg/blob/3fdc87371c72f7b4e1d91b85d5d8a9897c8867fc/.github/workflows/upload-release-to-plugin-repo.yml#L97-L161). You will see that the SVN command is executed in GitHub Actions.

**Comment by binarykitchen** [comment]
I'm impressed. This is so much better than other solutions I've seen.

Thanks for the pointer.
